src/main/java/uk/gov/hmcts/dts/fact/util/Filters.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.dts.fact.util;
+
+import uk.gov.hmcts.dts.fact.model.Contact;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+public final class Filters {
+
+    private Filters() {
+
+    }
+
+    public static List<String> filterDxNumber(List<Contact> contacts) {
+        List<Contact> dx = contacts
+            .stream()
+            .filter(c -> "DX".equals(c.getName()))
+            .collect(toList());
+        contacts.removeAll(dx);
+        return dx
+            .stream()
+            .map(Contact::getNumber)
+            .collect(toList());
```

Comment:
couldn't this just be implemented as:

```
        return contacts
            .stream()
            .filter(c -> !"DX".equals(c.getName()))
            .map(Contact::getNumber)
            .collect(toList());
```

Reply:
i think there should be 2 different methods, because this returns the array of dx numbers. we can have a method as you suggested to remove instead of using removeALL. but in this case, we returning the dx numbers, not a list without it.

Reply:
Ok - have refactored